syntax = "proto3";
package system_map;




import "easy_work_service_sdk/model/easy_work_service/app_system.proto";

/*
ListSystem请求
*/
message ListSystemRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页数据量
    */
    int32 pageSize = 2 ;
    /*
    应用系统类型
    */
    string category = 3 ;
    /*
    应用系统模糊匹配
    */
    string keyword = 4 ;
}

/*
ListSystem返回
*/
message ListSystemResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated easy_work_service.AppSystem list = 4 ;
}

/*
ListSystemApi返回
*/
message ListSystemResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    system_map.ListSystemResponse data = 4 ;
}

